lib/oidcstrategy.js

```diff
@@ -645,7 +646,7 @@ Strategy.prototype._validateIdToken = function validateIdToken(id_token, code, a
 
     // return jwt claims and jwt claims string
     var idTokenSegments = id_token.split('.');
-    var jwtClaimsStr = new Buffer(idTokenSegments[1], 'base64').toString();
+    var jwtClaimsStr = base64url(new Buffer(idTokenSegments[1], 'base64'));
```

Comment:
@lovemaths I think you should use base64url.decode. https://github.com/brianloveswords/base64url#base64urldecodeinput-string-encoding-string--utf8-string

What we want here is to decode the base64url encoded payload into string. Use something like this:
`base64url.decode(idTokenSegments[1])`
This will return the string in utf8 encoding.